sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -0,0 +1,29 @@
+[id="understanding-sandboxed-containers"]
+= Understanding OpenShift sandboxed containers
+include::modules/common-attributes.adoc[]
+
+:context: understanding-sandboxed-containers
+
+toc::[]
+
+[role="_abstract"]
+
+{sandboxed-containers-first} support for Red Hat OpenShift provides users with built-in support for running Kata Containers as an additional optional
```

Comment:
Should `Red Hat OpenShift` be `OpenShift Container Platform` (and thus `{product-title}`)? If so, you could likely omit `on {product-title}` from the end of the sentence.